templates/spago.dhall

```diff
@@ -1,6 +1,15 @@
 {-
 Welcome to a Spago project!
 You can edit this file as you like.
+
+Need help? See the following resources:
+- Spago project:        https://github.com/purescript/spago
+- Package-Sets project: https://github.com/purescript/package-sets
```

Comment:
```suggestion
```
I'd remove this link because I think it would distract more than be useful - we link to this anyways from the Spago docs in the relevant places